src/index.js

```diff
@@ -752,7 +752,7 @@ module.exports = {
 			// Return with the error as JSON object
 			res.setHeader("Content-type", "application/json; charset=utf-8");
 
-			const code = _.isNumber(err.code) ? err.code : 500;
+			const code = _.isNumber(err.code) && _.inRange(err.code, 400, 500) ? err.code : 500;
```

Comment:
Would be better a range between 400 & 599

Reply:
I'll change it